src/base/bittorrent/torrenthandle.h

```diff
@@ -129,6 +129,11 @@ namespace BitTorrent
         Error
     };
 
+    static uint qHash(TorrentState key, uint seed)
+    {
+        return ::qHash(static_cast<std::underlying_type_t<TorrentState>>(key), seed);
```

Comment:
move this to .cpp file and don't forget the `const` to variables.

Reply:
done